app/controllers/home_controller.rb

```diff
@@ -1,4 +1,10 @@
+# frozen_string_literal: true
+
 class HomeController < ApplicationController
   def index
+    unless current_user.status
+      sign_out
+      redirect_to sign_in_path
+    end
```

Comment:
if 
If I run the app without being logged in, I get this error, because there is no user in turn. 

![imagen](https://user-images.githubusercontent.com/12243331/107051053-97d26900-6791-11eb-814a-b741b5b2f9ce.png)

Reply:
Ready 👍